ui/src/org.ts

```diff
@@ -433,33 +429,35 @@ export async function fetchMembers(
     gnosisSafeAddress
   );
 
+  return {
+    threshold: response.threshold,
+    members: response.members.map(address => {
+      return { ethereumAddress: address };
+    }),
```

Comment:
The idea behind  making the `identity` field on `Member` optional was to indicate that we tried to resolve the link identity but it didn’t exist. Here a missing `identity` field indicates that we haven’t tried to resolve this yet. To avoid this ambiguity and to make the code overall simpler we could just have a list of addresses here.
```suggestion
    members: response.members
```

Reply:
Like this: 01d7a759c6de138673ccd20260305bcf2e1765e1?